api/config/config.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	Env        string `yaml:"env"`
 	Port       string `yaml:"port"`
 	Debug      bool   `yaml:"debug"`
+	mongoDbConnStr   string `yaml:"mongoDb"`
```

Comment:
maybe make this a Database {
       MongoDBConnStr string `yaml:"..."`
       DatabaseName  string ...
       UsersCollectionName ...
} `yaml:"database"`